spec/paths/payments@{id}.yaml

```
@@ -30,6 +31,7 @@ put:
   tags:
     - Payments
   summary: Create a payment with predefined ID
+  operationId: createPaymentWithPredefinedId
```

```suggestion
  operationId: createOrUpdatePayment
```
according to style of other items, but maybe better to use 'upsertPayment' (same for similar `createOrUpdate`)? To make operationId shorter?

---

yes, I was choosing between upsert (mongo style) and replace (mysql style). But these terms are so technical! How do you think?

---

I'm not sure that non-technical users will read specification, but I may be wrong

---

Me either. The API documentation is for API integration implementatin. It's written by thecnical people. Moreother, I don't think the operation names are useful for people who read the doc, the purpose of these names is not for documentation.